dms/views/directory.xml

```diff
@@ -0,0 +1,697 @@
+                                </tree>
+                            </field>
+                        </page>
+                        <page name="mail_alias" string="Email Alias">
+                            <group>
+                                <group>
+                                    <field
+                                        name="alias_contact"
+                                        string="Accept Emails from"
+                                    />
+                                    <field name="alias_process" />
+                                </group>
+                                <group>
+                                    <label for="alias_name" string="Email Alias" />
+                                    <div>
+                                        <field name="alias_name" class="oe_inline" />
+                                        <span class="oe_read_only">@</span>
+                                        <span class="oe_edit_only"> @ </span>
+                                        <field
+                                            name="alias_domain"
+                                            placeholder="mail.catchall.domain"
+                                            class="oe_inline"
+                                            readonly="1"
+                                        />
+                                    </div>
+                                    <field
+                                        name="alias_id"
+                                        readonly="1"
+                                        required="0"
+                                        groups="base.group_no_one"
+                                        string="Alias Setting"
+                                    />
+                                </group>
+                            </group>
+                        </page>
+                    </notebook>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" widget="mail_followers" />
+                    <field name="activity_ids" widget="mail_activity" />
+                    <field name="message_ids" widget="mail_thread" />
+                </div>
+            </form>
+        </field>
+    </record>
+    <record id="view_dms_directory_manager_form" model="ir.ui.view">
+        <field name="name">dms_directory.form</field>
+        <field name="model">dms.directory</field>
+        <field name="inherit_id" ref="view_dms_directory_form" />
+        <field name="groups_id" eval="[(6, 0, [ref('dms.group_dms_manager')])]" />
+        <field name="arch" type="xml">
+            <field name="is_root_directory" position="attributes">
+                <attribute name="invisible">0</attribute>
+            </field>
+            <field name="root_storage_id" position="attributes">
+                <attribute name="readonly">0</attribute>
```

Comment:
Maybe `default_root_storage_id` context value can be used here in order to prevent creating a directory with a different storage from a certain storage form

Reply:
Added